distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text.enso

```diff
@@ -55,15 +55,15 @@ type Text
          ACCENT). Therefore:
 
              (('É' . equals_ignore_case 'é') && ('é' . equals_ignore_case 'e\u0301')) == True
+    @locale Locale.default_widget
     equals_ignore_case : Text -> Locale -> Boolean
     equals_ignore_case self that locale=Locale.default =
         Text_Utils.equals_ignore_case self that locale.java_locale
 
     ## ADVANCED
-       PRIVATE
-       UNSTABLE
        Unifies the case of all letters in the text, generating a key which can be
        used to perform case-insensitive comparisons.
+    @locale Locale.default_widget
     to_case_insensitive_key : Locale -> Text
     to_case_insensitive_key self locale=Locale.default =
```

Comment:
I'm not completely sure if we should un`PRIVATE` this - as it is a bit of an implementation detail. But I appreciate it can be useful to users too.